packages/appium/lib/cli/extension.js

```diff
@@ -22,14 +22,12 @@ async function runExtensionCommand (args, type) {
   if (!extCmd) {
     throw new TypeError(`Cannot call ${type} command without a subcommand like 'install'`);
   }
-  let {json, suppressOutput, appiumHome} = args;
+  let {json, suppressOutput, appiumHome, appiumPluginHome} = args;
   if (suppressOutput) {
     json = true;
   }
   const logFn = (msg) => log(json, msg);
-  const ConfigClass = type === DRIVER_TYPE ? DriverConfig : PluginConfig;
-  const CommandClass = type === DRIVER_TYPE ? DriverCommand : PluginCommand;
-  const config = new ConfigClass(appiumHome, logFn);
+  const config = type === DRIVER_TYPE ? new DriverConfig(appiumHome, logFn) : new PluginConfig(appiumPluginHome || appiumHome, logFn); const CommandClass = type === DRIVER_TYPE ? DriverCommand : PluginCommand;
```

Comment:
this formatting is quite difficult to make sense of. can we make it more transparent what's happening?